README.md

```diff
@@ -735,16 +804,27 @@ The most flexible webhook GitHub offers is the `workflow_job` webhook, it includ
 This webhook should cover most people's needs, please experiment with this webhook first before considering the others.
 
 ```yaml
+apiVersion: actions.summerwind.dev/v1alpha1
 kind: RunnerDeployment
 metadata:
-   name: example-runners
+  name: example-runners
+  namespace: actions-runner-system
 spec:
   template:
     spec:
+      labels:
+        - self-hosted
```

Comment:
```suggestion
```

AFAIK, you don't need to explicitly set the `self-hosted` label as it's implied and forced by `actions/runner`/GitHub

Reply:
I think you are right, I was pretty sure it was not working before I added this line, but I just tested without it and it worked just as well.